packages/auth/src/api/index.test.ts

```diff
@@ -367,11 +367,11 @@ describe('api/_performApiRequest', () => {
     });
   });
 
-  context('edgcase error mapping', () => {
+  context('edge cage error mapping', () => {
```

Comment:
it's possible `edgcase` is a meaningful thing, but I'm assuming it isn't

Reply:
Suggest "edge case" instead of "cage."

Reply:
Sigh. I have no idea how I messed that one up.